src/__tests__/ParserComponent.test.tsx

```diff
@@ -265,6 +265,60 @@ describe('Csv FileEvents processed correctly\n', () => {
   });
 });
 
+describe('should accept valid csv file name with unusual' +
+    ' chars in file name', () => {
+  const props = {
+    prompt: 'test: ',
+    fileType: FileType.csv,
+  };
+
+
+  const onChangeMock = jest.fn();
+  const comp: any = mount(<ParserComponent
+    {...props}
+    onChange={onChangeMock}
+  />);
+
+  // clear the on change event mock and the enzyme component
+  beforeEach(() => {
+    onChangeMock.mockClear();
+    comp.setState({data: []});
+  });
+
+  it('file name with \\', async () => {
```

Comment:
Osgood was wanting some more docs about what this is really testing and what the expected output would be